fvtest/gc_api_test/TestIntrusiveList.cpp

```diff
@@ -0,0 +1,194 @@
+#include <OMR/IntrusiveList.hpp>
+#include <gtest/gtest.h>
+
+namespace OMR
+{
+
+struct LinkedInt;
+struct LinkedIntTraits;
+
+using LinkedIntListNode = IntrusiveListNode<LinkedInt>;
+
+static_assert(std::is_standard_layout<LinkedIntListNode>::value,
```

Comment:
is static assert available on all of our supported compilers?

Reply:
The compiler component use `statis_assert` so it's available on all supported compilers, presumably.

Reply:
(see https://github.com/eclipse/omr/blob/97818b41e1a29e687bcfd30a69f04870c343ae2a/compiler/README.md#build-info)

Reply:
Can you point to files in the compiler component that use it? There is lots of code in the compiler that does not compile on our minimum support compilers.

Reply:
Sure :slightly_smiling_face: 

https://github.com/eclipse/omr/blob/71f359066d5b6303ae78d81ce4c25c99f4066cff/compiler/env/TRMemory.hpp#L149

https://github.com/eclipse/omr/blob/71f359066d5b6303ae78d81ce4c25c99f4066cff/compiler/il/OMRDataTypes.hpp#L259-L264

https://github.com/eclipse/omr/blob/71f359066d5b6303ae78d81ce4c25c99f4066cff/compiler/il/OMRIL.cpp#L521-L582

https://github.com/eclipse/omr/blob/71f359066d5b6303ae78d81ce4c25c99f4066cff/compiler/il/ILProps.hpp#L126

https://github.com/eclipse/omr/blob/71f359066d5b6303ae78d81ce4c25c99f4066cff/compiler/codegen/OMRCodeGenerator.cpp#L138-L140

There are several more of these and most are in parts of the code that are fundamental to the compiler. :slightly_smiling_face:

Reply:
perfect! Thanks @Leonardo2718